docs/source/user_guide/gcm_based_inference/answering_causal_questions/attribute_distributional_changes.rst

```diff
@@ -3,51 +3,85 @@ Attributing Distributional Changes
 
 When attributing distribution changes, we answer the question:
 
-    What mechanism in my system changed between two sets of data?
+    **Given two datasets, which node behaves differently?**
```

Comment:
Is this really better? Note that we already use the word "mechanism" in our methods and language. So I believe it does make sense to refer to it here, no?

Reply:
It would definitely make sense, but seeing it from a perspective of someone who just wants to "quickly" look through the functionalities, I think this formulation helps more in understanding what the purpose of the method is. Otherwise, one first needs to understand what we mean by a  'mechanism' etc.

Reply:
Can we have both?  One for continuity across the documentation and one for the quick reader?  "What mechanism ....?  Or in other words, which node in my data behaves differently?"

Reply:
That's a good compromise, will change it.